app/views/service_providers/show.html.erb

```diff
@@ -222,3 +222,44 @@
   service_provider_path(service_provider),
   class: 'usa-button usa-button--outline float-left',
   method: :delete, data: { confirm: 'Are you sure you want to delete this app? This cannot be undone.' } %>
+
+
+<% if current_user.admin? %>
+  <div class="usa-accordion">
+    <button class="usa-accordion__button margin-top-105" aria-expanded="false" aria-controls="versions">
+      <h3 id="versionToggle" class="usa-accordion__heading">Version History (<%= service_provider.versions.count %>) </h3>
+    </button>
+    <div id="versions" class="usa-accordion__content">
+    <% service_provider.versions.reverse.each do |v| %>
+      <div class="event">
+        <h4 class="margin-0">Action: <%= v.event.capitalize %></h4>
+        <h5 class="margin-0">By: <%= v.whodunnit %></h5>
+        <h5 class="margin-0">At: <%= v.created_at %></h5>
+        <% if v.event != 'create' %>
```

Comment:
[question] is there a reason to leave out the initial create? probably helpful to know the starting point!

Reply:
My thought process there was:

* The create list is rather long
* We can go through changes to get back to the initial values
* If the list of versions is super long making reversing the changes to get to the original difficult, knowing what the original value was probably isn't useful

Reply:
that makes sense. i think there is possibly some utility in knowing the first iteration, but that's just a guess, so we can add it if we need it! thanks 🙏